test/footnote_test.exs

```diff
@@ -71,14 +71,11 @@ defmodule FootnoteTest do
              "    * List Item 2"
              ]
     {result, _} = Parser.parse(lines)
-    expected = [%Block.Para{lines: ["This is a footnote[^fn-1]"]},
-                %Block.FnDef{id: "fn-1", blocks: [%Block.Para{lines: ["line 1", "line 2"]},
-                                                  %Block.Para{lines: ["Para 2 line 1", "Para 2 line 2"]},
-                                                  %Block.List{blocks: [
-                                                      %Block.ListItem{blocks: [%Block.Para{lines: ["List Item 1", "  List Item 1 Cont"]}], spaced: false},
-                                                      %Block.ListItem{blocks: [%Block.Para{lines: ["List Item 2"]}], spaced: false}
-                                                      ]}
-                                                  ]}]
+    expected = [%Earmark.Block.Para{attrs: nil, lines: ["This is a footnote[^fn-1]"]}, 
```

Comment:
you are calling `alias Earmark.Block` so there's no need to use `Earmark.Block`. `Block` is ok